scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -258,6 +283,8 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
                                   should_build_tools=args.build_tools,
                                   build_config=args.config,
                                   seedlist_paths=[] if not args.seedlist_paths else args.seedlist_paths,
+                                  seedfile_paths=[] if not args.seedfile_paths else args.seedfile_paths,
+                                  level_names=[] if not args.level_names else args.level_names,
```

Comment:
We can forward the tool config argument through here.
Using a keyword parameter `tool_config=args.tool_config,`